packages/backend/src/managers/applicationManager.ts

```diff
@@ -157,6 +161,7 @@ export class ApplicationManager {
     await timeout(5000);
     await this.restartContainerWhenModelServiceIsUp(engineId, modelServiceEndpoint, container).catch(
       (error: unknown) => {
+        this.telemetry.logError('recipe.pull', { message: 'error monitoring endpoint', error: error });
```

Comment:
Event is not uniform here recipe id, recipe name and duration are not sent. I'm wondering if we should have a single point for sending this event (pullApplication) and handle exceptions there